Transliterate.cs

```diff
@@ -1,44 +1,64 @@
+// This sample requires C# 7.1 or later for async/await.
+
 using System;
 using System.Net.Http;
 using System.Text;
+using System.Threading.Tasks;
+// Install Newtonsoft.Json with NuGet
 using Newtonsoft.Json;
```

Comment:
you can remove this now, right?

Reply:
Hey @alexhcheng, the sample breaks when I remove `using System.Threading.Tasks`. I start getting this error: 

![Screen Shot 2019-05-23 at 2 49 12 PM](https://user-images.githubusercontent.com/42188191/58288976-f495b200-7d69-11e9-949b-b732b14a5768.png)